k8s-cronjobs.tf

```diff
@@ -0,0 +1,82 @@
+/**
+ * Copyright (C) 2018-2020 Expedia Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ */
+
+resource "kubernetes_cron_job" "apiary_inventory_repair" {
+  count = (var.s3_enable_inventory && var.hms_instance_type == "k8s") ? 1 : 0
+  metadata {
+    name      = "s3-inventory-repair"
+    namespace = "metastore"
+
+    labels = {
+      name = "s3-inventory-repair"
+    }
+  }
+
+  spec {
+    concurrency_policy = "Replace"
+    failed_jobs_history_limit = 5
+    schedule = var.s3_inventory_update_schedule
+
+    job_template {
+      metadata {}
+      spec {
+        template {
+          metadata {
+            labels = {
+              name = "s3-inventory-repair"
+            }
+            annotations = {
+              "iam.amazonaws.com/role" = aws_iam_role.apiary_hms_readwrite.name
```

Comment:
@rpoluri - I'm not sure if this is the correct role.  It worked in `egdp-dev`, but maybe it should be different?